esp/smc/SMCLib/TpCommon.cpp

```diff
@@ -170,6 +170,13 @@ extern TPWRAPPER_API bool validateDropZoneHostAndPath(const char* dropZoneName,
     Owned<IPropertyTree> plane = getDropZonePlane(dropZoneName);
     if (nullptr == plane)
         return false;
+    if (isContainerized() && streq("localhost", hostToCheck))
```

Comment:
When a host is "localhost", a LZ name can be found by findDropZonePlane() if a path is valid. I am not sure we should throw the 'makeStringException(ECLWATCH_INVALID_INPUT, "No dropzone or host not provided.")' here.

Reply:
The only reason host is ever "localhost" in a containerized environment, is because another service has used it as a placeholder, it will never be a actual host in plane with hosts.